mne/viz/_3d.py

```diff
@@ -559,9 +616,25 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
     surfs = dict()
     if head:
         subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
-        head_surf = _get_head_surface(subject, source=source,
-                                      subjects_dir=subjects_dir,
-                                      raise_error=False)
+        if bem is not None:
+            if isinstance(bem, ConductorModel):
+                if is_sphere:
+                    head_surf = _complete_sphere_surf(bem, 3, 4)
+                else:
+                    head_surf = _bem_find_surface(bem,
+                                                  FIFF.FIFFV_BEM_SURF_ID_HEAD)
+                    complete_surface_info(head_surf, copy=False, verbose=False)
+            elif bem is not None:  # list of dict
+                for this_surf in bem:
+                    if this_surf['id'] == FIFF.FIFFV_BEM_SURF_ID_HEAD:
+                        head_surf = this_surf
+                        break
+                else:
+                    raise ValueError('Could not find the surface for head.')
+        else:  # BEM solution
```

Comment:
I don't understand this comment -- this conditional is executed when `bem is None`